### DISCUSSION TITLE
update than script

### DISCUSSION THREAD

**Discussion by TungJeny** [nguyentrungtung/script_facebook#11] [Q&A] [answered]
i wants given than tools auto clear facebook

**Comment by nguyentrungtung** [comment, marked as answer]
yes! sure. I will do it in the near future! thanks you so much!

**Reply by TungJeny** [reply]
thanks you!

**Comment by TungJeny** [comment]
i wants a tools login facebook script js login not influent checkpoint

**Reply by nguyentrungtung** [reply]
i have code index.js login facebook not influent checkpoint in repository
